container/clip.go

```diff
@@ -0,0 +1,70 @@
+package container
+
+import (
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/widget"
+)
+
+// Declare conformity with CanvasObject interface
+var _ fyne.CanvasObject = (*Clip)(nil)
```

Comment:
shouldn't this be asserting against the `fyne.Widget` interface, since it's a widget, and the fact that it's clipping is actually looked up based on the renderer in the driver? (In other words, it would not work if container.Clip were "merely" a fyne.CanvasObject)

Reply:
Fixed